modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/AddFormInstanceRecordMVCActionCommand.java

```diff
@@ -146,25 +146,26 @@ protected void doProcessAction(
 		DDMFormInstanceSettings ddmFormInstanceSettings =
 			ddmFormInstance.getSettingsModel();
 
+		String successRedirectURL = ddmFormInstanceSettings.redirectURL();
+
 		String redirectURL = ParamUtil.getString(
-			actionRequest, "redirect", ddmFormInstanceSettings.redirectURL());
+			actionRequest, "redirect", successRedirectURL);
```

Comment:
@rijema Please move this to be right before the `sendRedirect(actionRequest, actionResponse, redirectURL);`, as it is only used there.

Reply:
Done.